roles/openshift_master/templates/master.yaml.v1.j2

```diff
@@ -8,6 +8,12 @@ assetConfig:
   logoutURL: ""
   masterPublicURL: {{ openshift.master.public_api_url }}
   publicURL: {{ openshift.master.public_console_url }}/
+{% if openshift.master.logging_public_url %}
```

Comment:
Hey @git001, changing these conditionals to something like this should get our tests to pass.

```
{% if 'logging_public_url' in openshift.master %}
```

Failure was:

```
TASK: [openshift_master | Create master config] *******************************
fatal: [10.8.52.78] => {'msg': "AnsibleUndefinedVariable: One or more undefined variables: 'dict object' has no attribute 'logging_public_url'", 'failed': True}
```